bootstrap/cmd/bootstrap/app/kfctlClient.go

```diff
@@ -21,6 +22,13 @@ type KfctlClient struct {
 	getEndpoint    endpoint.Endpoint
 }
 
+type KfctlService interface {
```

Comment:
Please add a TODO per discussion that this should be replaced by a versioned API

KfctlServiceV1Alpha1
KfctlServiceV1Beta1
KfctlServiceV1

That would use the corresponding versioned  KFDef spec as arguments.